guidelines/index.html

```diff
@@ -1432,30 +1432,30 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
+																																																																																													<div class="note" role="note" id="issue-container-generatedID-29"><div role="heading" class="note-title marker" id="h-note-29" aria-level="5"><span>注記 1</span></div><p class="">ユーザインタフェースコンポーネント又はサブコンポーネントとして認識されるもの (輪郭又は大きさを決定づけるもの) は、その視覚的<a href="#dfn-presentation" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-presentation-3" title="利用者が知覚できる形式でコンテンツをレンダリングすること。">提示</a>に依存する。視覚的提示には、コンポーネントの可視<a href="#dfn-content" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-content-1" title="コンテンツの構造、提示、及びインタラクションを定義するコード又はマークアップも含めて、ユーザエージェントによって利用者に伝達される情報及び感覚的な体験。">コンテンツ</a>、ボーダー、及びコンポーネント固有の背景が含まれる。コンポーネントのコンテンツ、背景、又はボーダーの外側にある、影及び光の効果は含まれない。</p></div>
 
-    <div class="note" role="note" id="issue-container-generatedID-30"><div role="heading" class="note-title marker" id="h-note-30" aria-level="5"><span>注記 2</span></div><p class="">Examples of sub-components that may receive a focus indicator are menu items in an opened drop-down menu, or focusable cells in a grid.</p></div>
+    <div class="note" role="note" id="issue-container-generatedID-30"><div role="heading" class="note-title marker" id="h-note-30" aria-level="5"><span>注記 2</span></div><p class="">フォーカスインジケータを受け取ることができるサブコンポーネントの例としては、開いたドロップダウンメニューの中のメニュー項目、又はグリッド内のフォーカス可能なセル、などがある。</p></div>
 
-    <div class="note" role="note" id="issue-container-generatedID-31"><div role="heading" class="note-title marker" id="h-note-31" aria-level="5"><span>注記 3</span></div><p class="">Contrast calculations can be based on colors defined within the <a href="#dfn-technologies" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-technologies-1" title="ユーザエージェントがどのようにレンダリング、再生、又は実行するかを符号化するメカニズム。">technology</a> (such as HTML, CSS and SVG). Pixels modified by user agent resolution enhancements and anti-aliasing can be ignored.</p></div>
+																																																																																													<div class="note" role="note" id="issue-container-generatedID-31"><div role="heading" class="note-title marker" id="h-note-31" aria-level="5"><span>注記 3</span></div><p class="">コントラストの計算は、(HTML、CSS、及び SVG といった) <a href="#dfn-technologies" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-technologies-1" title="ユーザエージェントがどのようにレンダリング、再生、又は実行するかを符号化するメカニズム。">技術</a>の中で定義された色を基に、行うことができる。その際、ユーザエージェントの解像度強化及びアンチエイリアスによって変更されたピクセルは、無視することができる。</p></div>
```

Comment:
計算は～計算できる　でちょっと重複しますがどうでしょう。
user agent resolution enhancementsは悩ましいですね…これどの機能を指すんですかね。
```suggestion
    <div class="note" role="note" id="issue-container-generatedID-31"><div role="heading" class="note-title marker" id="h-note-31" aria-level="5"><span>注記 3</span></div><p class="">コントラストの計算は、(HTML、CSS、SVG のような) <a href="#dfn-technologies" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-technologies-1" title="ユーザエージェントがどのようにレンダリング、再生、又は実行するかを符号化するメカニズム。">技術</a>の中で定義された色を基に計算できる。その際、ユーザエージェントの解像度強化及びアンチエイリアスによって変更されたピクセルは無視できる。</p></div>
```

Reply:
resolution enhancement はピクセル補完による解像度向上のことだと思います。
たとえばこういうやつとか: https://waifu2x.org/ja/

「解像度向上」とか「高解像度化」とかですかね?

Reply:
以下のように修正しようと思います。

> コントラストは、(HTML、CSS、及び SVG **のような**) 技術の中で定義された色を基に**計算できる**。その際、ユーザエージェント**による解像度向上**及びアンチエイリアスによって変更されたピクセルは**無視できる**。

Reply:
以下の commit にて修正しました。
[379969b](https://github.com/waic/wcag22/pull/37/commits/379969bc1fd17c738efc8b13c8602e3aa2986475)